src/app/pages/openstax-tutor/learn-more.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
+
+function ButtonBox({num, model}) {
+    const url = model[`section7CtaLink${num}`];
+    const text = model[`section7CtaText${num}`];
+    const description = model[`section7CtaBlurb${num}`];
+
+    return (
+        <div class="button-box">
+            <a class="btn" href={url}>{text}</a>
+            <div class="smaller">{description}</div>
+        </div>
+    );
+}
+
+export default function LearnMore({model}) {
+    return (
+        <section id="learn-more">
+            <div class="boxed">
+                <h1>{model.section7Heading}</h1>
+            </div>
+            <div class="text-content">
+                <div>{model.section7Subheading}</div>
+                <div class="buttons-row">
+                    {
+                        [1, 2].map((num) => <ButtonBox num={num} key={num} model={model} />)
+                    }
+                </div>
+            </div>
+        </section>
+    );
+}
```

Comment:
just curious but what is the `section7` that's mentioned several places in this page?

Reply:
In the CMS the data for each section is grouped by naming the data items with a prefix. The data for the Learn More section is prefixed section7. The FAQ is section6, etc.